### PR TITLE
List Transformation Tests

### PR DESCRIPTION
**Pull request by nitro-marky**
A selection of tests to make sure that the function is working. If there is one you want added let me know

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ program
     `)
     try {
       const res: Process.RawPayload[] = await getAll(mapOptions(options))
+      console.log(res)
```
Comment (mattdean-digicatapult) [inline src/index.ts:60]: remove `console.log`

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]
One tiny change please

File: src/index.ts (hunk 1 above)
Comment (nitro-marky) [reply src/index.ts:60]: Damn, missed that one

Comment (nitro-marky) [reply src/index.ts:60]: Oh not my one

**Review by mattdean-digicatapult — APPROVED** [review]
lgtm

**Pull request merged**